src/tools/gophish_import.py

```diff
@@ -281,12 +282,12 @@ def main() -> None:
         with open(args["ASSESSMENT_FILE"]) as json_file:
             assessment = json.load(json_file)
     except FileNotFoundError as e:
-        logging.error(f"{e}\n")
+        logging.error("%s", e)
         # Stop logging and clean up
         logging.shutdown()
         sys.exit(1)
     except PermissionError as e:
-        logging.error(f"{e}\n")
+        logging.error("%s", e)
```

Comment:
Why not use [`logging.exception()`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) here instead of recreating the wheel?

Reply:
See 44e24edb9ac96eb42759bdea8a9229563811ca59

Reply:
Also been updated here 3ee423033fefa9873fd13190c0e6edb575cd0080